autoPyTorch/api/base_task.py

```diff
@@ -1316,15 +1317,17 @@ def get_models_with_weights(self) -> List:
             self._load_models()
 
         assert self.ensemble_ is not None
-        return self.ensemble_.get_models_with_weights(self.models_)
+        models_with_weights: List = self.ensemble_.get_models_with_weights(self.models_)
```

Comment:
This change I do not full understand :)... shouldn't the get_models_with_weights return a list?

If this has to happen, can you do `List[float, Pipeline]` ?

Reply:
yes but mypy was raising a no-any-return error when I had no annotation but I have added your suggestion